pandas/tests/groupby/test_groupby.py

```diff
@@ -1736,3 +1736,18 @@ def test_groupby_multiindex_series_keys_len_equal_group_axis():
     expected = pd.Series([3], index=ei)
 
     assert_series_equal(result, expected)
+
+
+def test_groupby_groups_in_BaseGrouper():
+    # GH 26326
```

Comment:
can you give a 1-line description of what this is testing

Reply:
sure. will do so in the next commit